kernel_gateway/services/activity/__init__.py

```diff
@@ -0,0 +1,82 @@
+# Copyright (c) Jupyter Development Team.
```

Comment:
A bit weird to have this in the __init__ that gets ref'ed form the handler in the same package. How about an ActivityManager in keeping with the rest of the jupyter setup?